tempesta_fw/t/functional/testers/stress.py

```diff
@@ -121,7 +128,7 @@ def assert_tempesta(self):
         if self.tfw_msg_errors:
             return
 
-        self.assertTrue(self.tempesta.stats.cl_msg_other_errors <= 0,
+        self.assertTrue(self.tempesta.stats.cl_msg_other_errors == self.bce,
```

Comment:
Same here, should be only in derived class.

Reply:
No, if we put this to derived class, we will still have false FAIL in many tests if have low request_keepalive